doc/maintenance/development_versions.rst

```diff
@@ -0,0 +1,46 @@
+.. _development_versions:
+
+How to Maintain Package Versions
+================================
+
+Sometimes, typically when proposing a change (e.g., modifying the Backend
+↔ Frontend protocol, or adding new methods into the
+``python3-copr-common`` package that other packages depend on), we need to
+specify version constraints between our components to avoid
+misinstallations.  Below are a few documented scenarios.
```

Comment:
There is one more scenario which is changes in `python3-copr`. If it's exactly the same like `python3-copr-common` we should explicitly say that.